### ISSUE TITLE
Install issues with mayavi and "conda install igl"

### ISSUE THREAD

**Issue by burhop** [wilko339/MetaStruct#3]
Hi Toby,
I didn't have Ananconda so I had to install that first (this one in particular https://repo.anaconda.com/miniconda/Miniconda3-py39_4.10.3-Windows-x86_64.exe ) .

The script then had a problem with building the mayavi wheel (see log)

- "conda install igl" - Gave an error. I changed to "conda install -c conda-forge igl" which worked. I've attached a log of the session
-  below (note: WIndows 11 running on Anaconda Popwershell)

after that I ran the sphere_example() with no problem

[installLog.txt](https://github.com/wilko339/MetaStruct/files/7919404/installLog.txt)
.

**Comment by wilko339** [comment]
Yes I've consistently had errors thrown from mayavi when installing, but this doesn't seem to impact the use of mayavi for visualisation. Worth looking into for sure, could be a mayavi thing.
